baselines/teams.md

```diff
@@ -563,17 +620,24 @@ Docs](https://docs.microsoft.com/en-us/microsoftteams/teams-live-events/live-eve
 
 ### Implementation
 
+All the settings in this section are configured in the **Microsoft Teams admin center**.
+
+#### MS.TEAMS.10.1v1, instructions:
+
 1.  Sign in to the **[Microsoft Teams admin
     center](https://admin.teams.microsoft.com).**
 
-2.  Select **Meetings** -\> **Live events policies**.
+2.  Select **Meetings** > **Live events policies**.
 
 3.  Select **Global (Org-wide default)**.
 
 4.  Set **Record an event** to **Organizer can record**.
 
 5.  Click **Save**.
 
+6.  If custom policies have been created, repeat these steps for each
+    policy, selecting the appropriate policy in step 3.
+
 ## 11. Data Loss Prevention
```

Comment:
Looks like MS.TEAMS.11.2v1 and MS.TEAMS.11.3v1 are exact copies of each other.

Also, it may still be too soon to change these, but do we have an issue to update the resource sections that point to Defender with a more precise pointer? I do think this change should be a part of Emerald.

For example, change
> The “Data Loss Prevention SHALL Be Enabled” section of the Defender for Office 365 Minimum Viable Secure Configuration Baseline.

to 
> [Section 4, Data Loss Prevention | Defender for Office 365 Minimum Viable Secure Configuration Baseline](https://github.com/cisagov/ScubaGear/blob/260-revise-teams-policy-24-based-on-the-feedback-and-internal-testing/baselines/defender.md#4-data-loss-prevention).

Reply:
I did not change the content in the last three policies since they are directed at defender. I agree with holding off until defender updates are in to have a more precise pointer and use the same language throughout the policy group and policies.